pkg/resourceprovider/provider.go

```diff
@@ -440,7 +481,7 @@ func (p *resourceProvider) runQuery(now pmodel.Time, queryInfo resourceQuery, re
 	// run the query
 	rawRes, err := p.prom.Query(context.Background(), now, query)
 	if err != nil {
-		return nil, fmt.Errorf("unable to execute query: %v", err)
+		return nil, err
```

Comment:
Why are we removing this?

Reply:
I didn't want to update the type of the err. If it was a custom error, then the function above would convert it to a normal error and it would lose the fields of statusCode and Query.